### PR TITLE
Update State Management and Security Objectives

### PR DESCRIPTION
**Pull request by ScotteRoberts**

### CODE REVIEW

File: learning_objectives.md [hunk 1]
```diff
@@ -285,15 +289,44 @@ Consultants should be able to:
...
+- Explain the following terms, including:
+
+  - Global State
+  - Provided State
+  - Actions
+  - Reducers
+  - Store
+  - Data Flow
+
+- Model the data flow of a basic redux store
```
Comment (iylim) [inline learning_objectives.md]: add more? Using Redux dev tools?

File: learning_objectives.md [hunk 2]
```diff
@@ -325,12 +358,34 @@ Consultants should be able to:
...
+  - OWASP and top 3 Attack vectors:
+    - Injection
+    - Broken Authentication
+    - Sensitive Data Exposure
+- Back-end:
+  - Implement an endpoint that registers users in an application
+  - Implement an endpoint that validates registered users and provides a security token (JWT or Cookie)
+  - Build a custom express middleware or install a package (like passport) that authenticates users in an application
+- Front-end:
+
```
Comment (iylim) [inline learning_objectives.md]: empty line?

File: learning_objectives.md [hunk 3]
```diff
@@ -325,12 +358,34 @@ Consultants should be able to:
...
+  - Store (on the browser) a security token that is used to identify an authenticated user during their
+
```
Comment (iylim) [inline learning_objectives.md]: empty line?

**Comment by ScotteRoberts** [comment]
@iylim I made the changes! (plus some extras) 😄

**Pull request merged**